Makefile

```diff
@@ -31,6 +31,13 @@ all build:
 	hack/build-go.sh $(WHAT)
 .PHONY: all build
 
+# Build Code with E2E tests artifact.
+# Example:
+# make e2e
+e2e:
+	hack/build-go.sh test/e2e/e2e.test
```

Comment:
Yes... that is what extended.test is.

On Thu, Feb 18, 2016 at 5:32 PM, jay vyas <notifications@github.com> wrote:

> In Makefile
> <https://github.com/openshift/origin/pull/7049#discussion_r53393899>:
>
> > @@ -31,6 +31,13 @@ all build:
> >  	hack/build-go.sh $(WHAT)
> >  .PHONY: all build
> >
> > +# Build Code with E2E tests artifact.
> > +# Example:
> > +# make e2e
> > +e2e:
> > +	hack/build-go.sh test/e2e/e2e.test
>
> The original requirement is to ship and end2end upstream binary as part of
> our packaging.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7049/files#r53393899>.
>